docs/production/deploying-to-cloud/secrets/AWS.mdx

```diff
@@ -1,29 +1,98 @@
 ---
 title: "AWS Secrets Manager"
 sidebarTitle: "AWS"
+description: "Attention Magers: This classified document contains vital intel on securing sensitive data within the AWS Secrets Manager vault. Study the protocols carefully to create and operationalize confidential secrets, granting you clearance to integrate them into your Mage data pipeline projects."
 ---
+![It's Classified](https://github.com/mage-ai/assets/blob/main/secrets_assets/Source_%20Giphy%20(4).gif?raw=True)
+### **Creating a New Secret in AWS Secrets Manager**
 
-### Python
+Before you can use AWS Secrets Manager with Mage, you need to create a new secret in the AWS Secrets Manager console. Follow these steps:
 
-To use secrets in Python code, read these [instructions](/design/data-loading#aws-secret-loader).
+1. Open the AWS Secrets Manager console: https://console.aws.amazon.com/secretsmanager/
```

Comment:
Embed the link with markdown instead of displaying the entire link outright. Sometimes, it makes sense to just show the url, but oftentimes, it's just cleaner to embed. You just use brackets around the text and then parens around the link, e.g. `[console](https://console.aws.amazon.com/secretsmanager/)`

Reply:
Added embedded link, tested and takes me to correct site from mintlify dev env.
<img width="556" alt="Screenshot 2024-06-20 at 1 22 06 PM" src="https://github.com/mage-ai/mage-ai/assets/92221031/eca55f2a-3a61-4be6-a54c-1a7610ce2a46">